tethysext/atcore/controllers/resource_workflows/results_views/map_workflow_results_view.py

```diff
@@ -140,6 +137,19 @@ def get_context(self, request, session, resource, context, model_db, workflow_id
 
             layer_groups.insert(0, results_layer_group)
 
+        if self.map_type == "cesium_map_view":
+            layers, entities = self.translate_layers_to_cesium(results_layers)
+            for layer in layers:
+                # Add layer to beginning the map's of layer list
+                map_view.layers.insert(0, layer)
+            for entity in entities:
+                # Add entity to beginning the map's of entity list
+                map_view.entities.insert(0, entity)
```

Comment:
Let's use list addition to preserve the order of `layers`:

```python
map_view.layers = layers + map_view.layers
map_view.entities = entities + map_view.entities
```

The loop will end up reversing the order of the layers and entities.

Reply:
That's how we had it before. I just don't want to change the behavior of the existing app.